packages/node-type-resolver/config/phpstan/static-reflection.neon

```diff
@@ -0,0 +1,15 @@
+parameters:
+    # see https://github.com/rectorphp/rector/issues/3490#issue-634342324
+    featureToggles:
+        disableRuntimeReflectionProvider: false
+
+services:
+    - Rector\NodeTypeResolver\Reflection\BetterReflection\RectorBetterReflectionSourceLocatorFactory
+    - Rector\NodeTypeResolver\Reflection\BetterReflection\SourceLocator\IntermediateSourceLocator
+    - Rector\NodeTypeResolver\Reflection\BetterReflection\SourceLocatorProvider\DynamicSourceLocator
+
+    # basically decorates native PHPStan source locator with a dynamic source locator that is also available in Rector DI
+    betterReflectionSourceLocator:
+        class: _HumbugBoxfac515c46e83\Roave\BetterReflection\SourceLocator\Type\SourceLocator
```

Comment:
@ondrejmirtes This is a pickle. Using your `roave\better-reflection` package version breaks here, as PHPStan requires the prefixed one. But prefix can change on any PHPStan release.

Any ideas how to avoid changing the prefix on each PHPStan release?

Reply:
@TomasVotruba This should help you, test the latest version please: https://github.com/phpstan/phpstan-src/commit/43ca4270faf7a51616111595563742501382ba8c

Reply:
Amazing, thanks for that :+1:

Reply:
It seems the autoloading is not able to find `Roave\BetterReflection\*` classes. Do I need to include it in autoload somewhere extra?

See: https://github.com/rectorphp/rector/pull/5665/checks?check_run_id=1969350846

![image](https://user-images.githubusercontent.com/924196/108990198-d1163e80-7696-11eb-83b7-334c7af8ce22.png)

Minimal step to reproduce (any command):

```bash
bin/rector show
```

Reply:
I'm using latest 3 hours old version: https://github.com/phpstan/phpstan/commit/17f9278613a868a1ad75faee5f6afa70b7e39ca9

Reply:
After this it will work for you: https://github.com/phpstan/phpstan/commit/daec3a68dca69b09ab4bb5244345fc3c8f480099

Just FYI I might rename the `Roave` namespace to `OndrejMirtes` or `PHPStan` just to avoid possible conflicts with this. I'll let you know if I do.

Reply:
Alright, so I did that: https://github.com/phpstan/phpstan-src/commit/06ef13e9c30626f657b2be10030cf0c857b1dc35

Please update your namespaces. I'm much more comfortable with this change than keeping Roave\BetterReflection unprefixed.

Reply:
Thank you :+1: 

I tried to `compsoer update` and run `bin/rector` but error is the same: https://github.com/rectorphp/rector/pull/5665/checks?check_run_id=1975382136

This is the config for PHPStan:
https://github.com/rectorphp/rector/pull/5665/files#diff-208af2a2bbfa3cab247f54b61881c39c9cb3c43b358b35effed69aefc9557458

Reply:
Oh yes I forgot about one part of the puzzle, will update it shortly 👍

Reply:
Here you go, it should work now: https://github.com/phpstan/phpstan/commit/f683544424e49055afa19505a0fbecea1921a2cc

Reply:
It works, thanks :+1: